### ISSUE TITLE
Staging Review finding: Follow the proper heading structure on the results page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98227]
## Status
**2024-12-23] [Fran]** Added to the Discharge Upgrade Wizard Phase 2 [ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19322)

---

_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Public Websites
**Product:** Discharge Upgrade Wizard
**Feature:** 

## Findings details
**VA.gov Experience Standard - issue:** [User can't determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @allison0034 (Design), @briandeconinck (Accessibility)

### Description

On the results page, if I answer that my discharge was related to post traumatic stress disorder (PTSD), the second step (Add supporting information) is an h3. Under that step there are bulleted headings (Military Record, Medical records, Buddy statements (also called a Statement in Support of Claim or VA Form 21-4138, and Testaments of achievements since your service). Those headings are also h3.

<details><summary>Link, screenshot or steps to recreate</summary>

![screen shot](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/80777-Allison.png?raw=true)

</details>

## Recommended action

Change "supporting information" to be an h2. You may also want to consider removing the bullets from the h3s under that. The bullets are indenting and taking up space and may not be necessary, however work with content and CAIA to confirm.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** [Typography](https://design.va.gov/foundation/typography)
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by FranECross** [comment]
Added to the Discharge Upgrade Wizard Phase 2 [ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19322)/review.

**Comment by FranECross** [comment]
The steps (main headings) have been change to be H2s, and the sub headings are then correct remaining H3s. Implemented, released and validated in production. ([Ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/20124)) Closing.
